packages/backend/src/projectAdapters/dbtBaseProjectAdapter.ts

```diff
@@ -64,6 +96,25 @@ export abstract class DbtBaseProjectAdapter implements ProjectAdapter {
         return this.rpcClient.runQuery(sql);
     }
 
+    static _isDbtModelMetadataValid(
```

Comment:
I always expect a boolean return from functions that start with `isX`.
I would have called this `validateDbtModelMetadata`